spec/helpers/user_helper_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+include ActiveSupport::Testing::TimeHelpers
+
+RSpec.describe UsersHelper do
+  describe '#user_age' do
+    context "when today's date is 2018-01-30 and user was born in 1994-08-21" do
```

Comment:
From your contexts/its I know, that the `user_age` helper:
- returns 23 if called 2018-01-30 with user born 1994-08-2,
- returns 24 if called 2018-01-30 with user born 1994-08-21,
- and I have no idea how it behaves when called on other dates and with different users

Of course, I can guess it as it's quite obvious now in that context, but ideally, specs should work as a documentation of your code. If you are not able to cover all possible cases with a separate contexts, then maybe that's not a way to go.
How about doing something as simple, as a single `it` with something along the lines "it returns the current age in years of a given user" - that describes exactly what that method does, and it can be quite easily tested.
If you want to make sure it works for more cases than you can simply create more than one user in the single spec and test it for all of them.
And if you want to test edge cases, you can wrap that single it in a context (e.g. "when user have a valid birthday set"), and add a separate one for the edge case (e.g. "when user have a birthday in the future").